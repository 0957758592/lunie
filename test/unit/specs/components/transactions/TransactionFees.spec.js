import { shallowMount } from "@vue/test-utils"
import TransactionFees from "src/components/transactions/TransactionFees"

describe(`TransactionFees`, () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(TransactionFees, {
      propsData: {
        time: new Date("2019-07-31"),
        block: 1234567,
        fees: {
          amount: "1234",
          denom: "uatom"
        }
      },
      stubs: [`router-link`]
    })
  })

  it(`has the expected html structure`, () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
