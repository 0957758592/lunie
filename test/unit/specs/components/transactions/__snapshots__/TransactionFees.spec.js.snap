// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransactionFees has the expected html structure 1`] = `
<div>
  <div>
    
    Network Fee: 
    
    <b>
      0.001234
    </b>
     
    <span>
      ATOM
    </span>
  </div>
   
  <div>
    <router-link-stub
      to="[object Object]"
    >
      Block #1234567 
    </router-link-stub>
    
    @ Jul 31st 2019 00:00:00
  
  </div>
</div>
`;
